src/gui/menu/description.py

```diff
@@ -74,8 +85,16 @@ def draw_slider_bar(self):
         pygame.draw.rect(self.display_surface, "grey", slide_bar_rect, 0, 4)
 
     def draw(self):
-        pygame.draw.rect(self.display_surface, "White", self.rect, 0, 4)
-        self.make_surface_transparent()
```

Comment:
Of course you'll get rendering artifacts if you remove this call...

Reply:
I moved the call on children classes